tests/ert_tests/ert3/data/test_record.py

```diff
@@ -46,7 +62,7 @@ def test_valid_record(data):
 )
 def test_invalid_record(data):
     with pytest.raises(pydantic.ValidationError):
-        ert.data.Record(data=data)
+        ert.data.NumericalRecord(data=data)
```

Comment:
Perhaps we should introduce a similar test for invalid blob records?

Reply:
Done